README.md

```diff
@@ -13,7 +13,8 @@ A repo to keep track of the Bolt Roadmap.
  
   - Clone repo
   - Run `composer install`
+  - Add your personal Github token to the `GITHUB_SECRET` environment variable in the `.env` file
```

Comment:
first of all, `cp .env.dist .env`

Reply:
Why would you need that? Isn't the `.env` file generated when running `composer install`?

Reply:
you are right :)

Reply:
👍